.expeditor/config.yml

```diff
@@ -17,6 +17,11 @@ docker_images:
 
 pipelines:
   - habitat/build
+  - omnibus/release
```

Comment:
We will also need to add the `trigger_pipeline:omnibus/release:` on merge action

Reply:
This will be added in a follow up PR.